tests/unit/backend/chalice/api_server/test_v1_auth.py

```diff
@@ -45,15 +45,17 @@ def test__auth_flow(self):
             self.assertEqual(args["client_id"], self.auth_config.client_id)
             self.assertEqual(args["response_type"], "code")
             self.assertTrue("/dp/v1/oauth2/callback" in args["redirect_uri"])
-            self.check_set_cookie_is_secure(response.headers["Set-Cookie"])
+            self.check_set_cookie_is_secure(response.headers["Set-Cookie"], ["HttpOnly", "Secure", "SameSite=Lax"])
 
             # follow redirect
             test_url = f"/dp/v1/oauth2/callback?code=fakecode&state={args['state']}"
             response = self.app.get(test_url, headers=dict(host="localhost", Cookie=response.headers["Set-Cookie"]))
             self.assertEqual(response.status_code, 302)
             self.assertEqual(response.headers["Location"], self.auth_config.redirect_to_frontend)
             self.assertTrue("Set-Cookie" in response.headers)
-            self.check_set_cookie_is_secure(response.headers["Set-Cookie"])
+            self.check_set_cookie_is_secure(
+                response.headers["Set-Cookie"], expected_flags=["HttpOnly", "Secure", "SameSite=Strict"]
```

Comment:
is this supposed to be `SameSite=Strict` or `Lax`? Thank you!

Reply:
this cookie should be `Strict`

Reply:
Got it thank you!